engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropNode.java

```diff
@@ -161,36 +150,15 @@ public void disableWireframe() {
     public void process() {
         PerformanceMonitor.startActivity("rendering/backdrop");
 
-        // Common Shader Parameters
-
-        skyMaterial.setFloat("viewingDistance", renderingConfig.getViewDistance().getChunkDistance().x * 8.0f, true);
-
-        skyMaterial.setFloat("daylight", backdropProvider.getDaylight(), true);
-        skyMaterial.setFloat("tick", worldRenderer.getMillisecondsSinceRenderingStart(), true);
-        skyMaterial.setFloat("sunlightValueAtPlayerPos", worldRenderer.getTimeSmoothedMainLightIntensity(), true);
-
-        cameraDir = activeCamera.getViewingDirection();
-        cameraPosition = activeCamera.getPosition();
-
-        skyMaterial.setFloat("swimming", activeCamera.isUnderWater() ? 1.0f : 0.0f, true);
-        skyMaterial.setFloat3("cameraPosition", cameraPosition.x, cameraPosition.y, cameraPosition.z, true);
-        skyMaterial.setFloat3("cameraDirection", cameraDir.x, cameraDir.y, cameraDir.z, true);
-        skyMaterial.setFloat3("cameraParameters", activeCamera.getzNear(), activeCamera.getzFar(), 0.0f, true);
+        // Shader Parameters
 
         sunDirection = backdropProvider.getSunDirection(false);
-        skyMaterial.setFloat3("sunVec", sunDirection.x, sunDirection.y, sunDirection.z, true);
-
-        skyMaterial.setFloat("time", worldProvider.getTime().getDays(), true);
```

Comment:
Yeah, in the final cleanup stages, I'm actually checking every Shader manually to remove the unused variables from the corresponding Node.